pkg/commands/compute/init.go

```diff
@@ -142,13 +152,18 @@ func (c *InitCommand) Exec(in io.Reader, out io.Writer) (err error) {
 		email = p.Email
 	}
 
-	name, desc, authors, err := c.PromptOrReturn(email, in, out)
-	if err != nil {
-		c.Globals.ErrLog.AddWithContext(err, map[string]any{
-			"Description": desc,
-			"Directory":   c.dir,
-		})
-		return err
+	var name string
+	var desc string
+	var authors []string
```

Comment:
NIT: maybe combine into single `var`...

```suggestion
	var (
		name string
		desc string
		authors []string
	)
```